web/src/games/reversi/locales/pt.json

```diff
@@ -17,7 +17,7 @@
     "green": "Verde",
     "players_turn": "Vez do {{ playerName }}",
     "place_piece": "Coloca peça",
-    "waiting_for_opponent": "Esperando por adiversário",
+    "waiting_for_opponent": "Esperando por adversário",
```

Comment:
noticed the typo

Reply:
Someone (me) needs to go back to primary school :P

Reply:
It was me, actually